desktop/conf.dist/hue.ini

```diff
@@ -239,9 +239,11 @@ http_500_debug_mode=false
 # Choose whether to allow multi tenancy or not.
 ## enable_organizations=false
 
-# Editor autocomplete timeout (ms) when fetching columns, fields, tables etc.
-# To disable this type of autocompletion set the value to 0.
-## editor_autocomplete_timeout=30000
+# Choose whether the editor autocomplete should gather suggestions from external source or not. The editor
+# autocomplete uses various sources for its suggestions, listing databases, tables, columns files etc. The results are
+# cached on the client (see cacheable_ttl) so the calls are kept to a minimum but if you prefer to disable these calls
+# all together from the editor set this to true.
+## disable_source_autocomplete=false
```

Comment:
Nit: Fix indentation

Reply:
For the whole file?

Reply:
Oops my bad, this file does not have indentation like pseudo one.